### DISCUSSION TITLE
ClojureFam

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#980] [Operations]
Clojurefam was a cohort-based clojure learning program that was maintained during a large part of 2020 at the beginning of the Athens project. It became dormant after @tangjeff0 who was supervising the cohorts needed to direct his attention to other parts of the project (correct me if I'm wrong here or if there was a more specific reason!). How can we set up Clojurefam to be more self sufficient or make it a better standalone self-learning experience? If you participated in clojurefam in the past consider sharing your thoughts and feedback so we can improve! 

Notes
- volunteer mentorship program (peer mentorship)
- entry points to contributing
- community spotlight of educational tutorials and material on athens youtube channel
- [ClojureFam](https://github.com/athensresearch/ClojureFam)
- https://github.com/marketplace/github-learning-lab

**Comment by juniusfree** [comment]
I'm part of the first cohorts. 🙋‍♂️ I'm the only one left.

Observations
1. My cohort has a broad range of skills/experience. One is a CS student, two are professionals without Clojure experience if I'm not mistaken, and in my case, I have almost zero experience in programming. I feel like it has an effect somehow.
2. One of the value prop of Clojurefam, initially, is having a regular mentor. Most learners might have expected that but it didn't happen.
3. I'm not sure why but the one who's managing it at the start becomes inactive in the community.
4. Of course, we can't expect everyone to continue. There might be some more important reasons why they quit.

Why I stayed (There are some days where I'm the only one who's posting even if the community overall is inactive.)
1. I'm just tired of not learning to code after failing several times in the past 10 years.
2. Clojure and functional programming are more appealing to me. (e.g. https://www.edx.org/course/how-to-code-simple-data)
3. The domain of PKM is growing and interesting. This is important because I'm not just learning to code but I can also see how coding solves real problems.
4. I can contribute even some small fixes and @tangjeff0 is patient to give feedback to my PR even if he can do it faster. (I'm just fortunate that I'm the only beginner. Of course, this is not scalable.)
5. I'm inspired by people who are way smarter than me.
6. I just want to be a beta user because Roam is expensive. :)

What other communities can we take inspiration from?
1. https://eriktorenberg.substack.com/p/whats-on-deck-for-on-deck
2. https://1729.com/a-newsletter-that-pays-you/

Some ideas:
1. partner with other Clojure communities that specifically focus on teaching Clojure (e.g. https://practicalli.github.io/ or https://lambdaisland.com/)

**Comment by jsmorabito** [comment]
https://github.com/alaq/learning-clojure-in-public/blob/master/review.md some great feedback here from @alaq

**Reply by alaq** [reply]
Wow, I’m glad you found it and that it was useful!

One more note on ClojureFam, I think at this point I didn’t contribute to Athens more after the program because of three reasons:
- tooling, I never felt happy with my setup and I think we can be more helpful to beginners there
- I picked up hard issues (to me) and eventually lost faith 
- I got sidetracked with other projects, writing in languages where I was actually productive. Even my little reagent side project during ClojureFam never reached completion because I didn’t know how to solve my issues.
